CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ The changes are relative to the previous release, unless the baseline is specifi
   AVIF file has an ImageRotation property with angle set to 1 or 3, has no
   ImageMirror property, and carries an Exif chunk. Note that Exif orientation is
   usually ignored in PNG files, so this mainly impacts JPEG files.
+* Always forward Unspecified (2) CICP color primaries, transfer characteristics,
+  and matrix coefficients to the AV1 encoder. Rely on the 'colr' box instead.
```

Comment:
Just FYI: I uploaded an AVIF file to the https://gpac.github.io/ComplianceWarden-wasm/ and it does not report any new errors. (It only reports a known error: https://github.com/AOMediaCodec/libavif/issues/2514.)